packages/mui-material/src/Link/Link.d.ts

```diff
@@ -58,8 +58,10 @@ export interface LinkTypeMap<P = {}, D extends React.ElementType = 'a'> {
  */
 declare const Link: OverridableComponent<LinkTypeMap>;
 
-export type LinkBaseProps = Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, 'color'> &
-  DistributiveOmit<TypographyProps, 'children' | 'component' | 'color' | 'ref' | 'variant'>;
+export type LinkBaseProps = DistributiveOmit<
+  TypographyOwnProps,
+  'children' | 'component' | 'color' | 'ref' | 'variant' | 'classes'
```

Comment:
There are no `component` and `ref` properties in `TypographyOwnProps` type:
```suggestion
  'children' | 'color' | 'variant' | 'classes'
```

Reply:
fixed here https://github.com/mui/material-ui/pull/35924/commits/a584afb399cc564b4647409aa5e693bacdabb9b4